src/NUnitConsole/nunit3-console.tests/CommandLineTests.cs

```diff
@@ -529,6 +529,8 @@ public void ShouldSetTeamCityFlagAccordingToArgsAndDefauls(bool hasTeamcityInCmd
         public void ShouldNotFailOnEmptyLine()
         {
             var testListPath = Path.Combine(TestContext.CurrentContext.TestDirectory, "TestListWithEmptyLine.tst");
+            // Not copying this test file into releases
+            Assume.That(testListPath, Does.Exist);
```

Comment:
We could add this to future release ZIPS easily enough, I just didn't want to deal with it in the middle of this release.